From 2432a8d40196f1be95c4a87950b9543c8331b2e7 Mon Sep 17 00:00:00 2001
From: Brad Love <brad@nextdimension.cc>
Date: Fri, 17 Mar 2017 01:37:24 +0000
Subject: [PATCH 5/7] maintainer and git url update

---
 debian.master/control.stub.in | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/debian.master/control.stub.in b/debian.master/control.stub.in
index 9f4f817..7504ae7 100644
--- a/debian.master/control.stub.in
+++ b/debian.master/control.stub.in
@@ -1,7 +1,7 @@
 Source: linux
 Section: devel
 Priority: optional
-Maintainer: Ubuntu Kernel Team <kernel-team@lists.ubuntu.com>
+Maintainer: Brad Love (Hauppauge) <brad@nextdimension.cc>
 Standards-Version: 3.9.4.0
 Build-Depends:
  debhelper (>= 5),
@@ -44,7 +44,8 @@ Build-Depends-Indep:
  bzip2 <!stage1>,
  sharutils <!stage1>,
  asciidoc <!stage1>,
-Vcs-Git: git://git.launchpad.net/~ubuntu-kernel/ubuntu/+source/linux/+git/xenial
+Vcs-Git: git://github.com/~b-rad-NDi/
+Vcs-Browser: https://github.com/~b-rad-NDi/
 XS-Testsuite: autopkgtest
 #XS-Testsuite-Depends: gcc-4.7 binutils
 
-- 
2.7.4

