From 57dda17fa467d00faea64c2ca4e3f00fa2a60222 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Wed, 24 May 2017 16:11:46 -0500
Subject: [PATCH 6/6] Changelog

---
 debian.master/changelog | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index fe86ee8..73290a4 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,29 @@
+linux (4.4.0-78201705241611.0+mediatree+hauppauge) xenial; urgency=low
+
+  * Ubuntu kernel git clone
+    - 4.4.0-78.99 rev 156e00060fbc5f735530bfdda36bab8b3d58ff71
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2017-04-20-20-54.patch
+    - 0002-Apply-build-fixes-to-media-tree.patch
+    - 0003-Add-new-media-tree-kernel-config.patch
+
+  * Ubuntu mainline build fixes
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+
+  * Packaging patches
+    - 0005-Maintainer-changelog-git-url-updates.patch
+
+  * Additional patches
+    - 0006-lgdt3306a-i2c-mux-backport.patch
+    - 0007-Hauppauge-WinTV-DualHD-DVB-ATSC-second-tuner-support.patch
+    - 0008-em28xx-bulk-transfer-implementation-fix.patch
+    - 0009-usb-packet-size-tweaks.patch
+    - 0010-lgdt3306a-add-QAM-AUTO.patch
+    - 0011-Add-Hauppauge-bulk-models-to-driver-table.patch
+
+ -- Fake User <hidden@email.co>  Wed, 24 May 2017 16:11:46 -0500
+
 linux (4.4.0-78.99) xenial; urgency=low
 
   * linux: 4.4.0-78.99 -proposed tracker (LP: #1686645)
-- 
2.7.4

