From aaef5d0d01ffecc72052bac2f866fbc7312e760b Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Wed, 8 Mar 2023 16:55:19 -0600
Subject: [PATCH 4/6] Mainline Ubuntu build fixes

---
 tools/perf/util/bpf-utils.c | 43 +++++++++----------------------------
 1 file changed, 10 insertions(+), 33 deletions(-)

diff --git a/tools/perf/util/bpf-utils.c b/tools/perf/util/bpf-utils.c
index 5a66dc8594aa..64a558344696 100644
--- a/tools/perf/util/bpf-utils.c
+++ b/tools/perf/util/bpf-utils.c
@@ -115,7 +115,7 @@ get_bpf_prog_info_linear(int fd, __u64 arrays)
 	__u32 info_len = sizeof(info);
 	__u32 data_len = 0;
 	int i, err;
-	__u8 *ptr;
+	void *ptr;
 
 	if (arrays >> PERF_BPIL_LAST_ARRAY)
 		return ERR_PTR(-EINVAL);
@@ -126,8 +126,6 @@ get_bpf_prog_info_linear(int fd, __u64 arrays)
 		pr_debug("can't get prog info: %s", strerror(errno));
 		return ERR_PTR(-EFAULT);
 	}
-	if (info.type >= __MAX_BPF_PROG_TYPE)
-		pr_debug("%s:%d: unexpected program type %u\n", __func__, __LINE__, info.type);
 
 	/* step 2: calculate total size of all arrays */
 	for (i = PERF_BPIL_FIRST_ARRAY; i < PERF_BPIL_LAST_ARRAY; ++i) {
@@ -175,8 +173,6 @@ get_bpf_prog_info_linear(int fd, __u64 arrays)
 					     desc->count_offset, count);
 		bpf_prog_info_set_offset_u32(&info_linear->info,
 					     desc->size_offset, size);
-		assert(ptr >= info_linear->data);
-		assert(ptr < &info_linear->data[data_len]);
 		bpf_prog_info_set_offset_u64(&info_linear->info,
 					     desc->array_offset,
 					     ptr_to_u64(ptr));
@@ -190,45 +186,26 @@ get_bpf_prog_info_linear(int fd, __u64 arrays)
 		free(info_linear);
 		return ERR_PTR(-EFAULT);
 	}
-	if (info_linear->info.type >= __MAX_BPF_PROG_TYPE) {
-		pr_debug("%s:%d: unexpected program type %u\n",
-			 __func__, __LINE__, info_linear->info.type);
-	}
 
 	/* step 6: verify the data */
-	ptr = info_linear->data;
 	for (i = PERF_BPIL_FIRST_ARRAY; i < PERF_BPIL_LAST_ARRAY; ++i) {
 		const struct bpil_array_desc *desc = &bpil_array_desc[i];
-		__u32 count1, count2, size1, size2;
-		__u64 ptr2;
+		__u32 v1, v2;
 
 		if ((arrays & (1UL << i)) == 0)
 			continue;
 
-		count1 = bpf_prog_info_read_offset_u32(&info, desc->count_offset);
-		count2 = bpf_prog_info_read_offset_u32(&info_linear->info,
+		v1 = bpf_prog_info_read_offset_u32(&info, desc->count_offset);
+		v2 = bpf_prog_info_read_offset_u32(&info_linear->info,
 						   desc->count_offset);
-		if (count1 != count2) {
-			pr_warning("%s: mismatch in element count %u vs %u\n", __func__, count1, count2);
-			free(info_linear);
-			return ERR_PTR(-ERANGE);
-		}
+		if (v1 != v2)
+			pr_warning("%s: mismatch in element count\n", __func__);
 
-		size1 = bpf_prog_info_read_offset_u32(&info, desc->size_offset);
-		size2 = bpf_prog_info_read_offset_u32(&info_linear->info,
+		v1 = bpf_prog_info_read_offset_u32(&info, desc->size_offset);
+		v2 = bpf_prog_info_read_offset_u32(&info_linear->info,
 						   desc->size_offset);
-		if (size1 != size2) {
-			pr_warning("%s: mismatch in rec size %u vs %u\n", __func__, size1, size2);
-			free(info_linear);
-			return ERR_PTR(-ERANGE);
-		}
-		ptr2 = bpf_prog_info_read_offset_u64(&info_linear->info, desc->array_offset);
-		if (ptr_to_u64(ptr) != ptr2) {
-			pr_warning("%s: mismatch in array %p vs %llx\n", __func__, ptr, ptr2);
-			free(info_linear);
-			return ERR_PTR(-ERANGE);
-		}
-		ptr += roundup(count1 * size1, sizeof(__u64));
+		if (v1 != v2)
+			pr_warning("%s: mismatch in rec size\n", __func__);
 	}
 
 	/* step 7: update info_len and data_len */
-- 
2.43.0

