From d40ae565837504352dd7a95cee3b5a6ff5a7f1a5 Mon Sep 17 00:00:00 2001
From: Bradford Love <hidden@email.co>
Date: Fri, 17 Jan 2025 14:26:45 -0600
Subject: [PATCH 5/6] Add new media tree kernel config

---
 debian.master/config/annotations | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/debian.master/config/annotations b/debian.master/config/annotations
index c18593ca8481..5e819ccc1a31 100644
--- a/debian.master/config/annotations
+++ b/debian.master/config/annotations
@@ -3,6 +3,8 @@
 # ARCH: amd64 arm64 armhf ppc64el riscv64 s390x
 # FLAVOUR: amd64-generic arm64-generic arm64-generic-64k armhf-generic ppc64el-generic riscv64-generic s390x-generic
 
+#CONFIG_GCC_ASM_GOTO_OUTPUT_WORKAROUND		policy<{'amd64': 'y', 'arm64': 'y', 'armhf': 'y'}>
+
 CONFIG_ACCESSIBILITY                            policy<{'amd64': 'y', 'arm64': 'y', 'armhf': 'y', 'ppc64el': 'y', 'riscv64': 'y', 's390x': 'n'}>
 CONFIG_ACCESSIBILITY                            note<'LP: #1967702'>
 
@@ -4507,7 +4509,7 @@ CONFIG_DVB_LNBP22                               policy<{'amd64': 'm', 'arm64': '
 CONFIG_DVB_M88DS3103                            policy<{'amd64': 'm', 'arm64': 'm', 'armhf': 'm', 'ppc64el': 'm', 'riscv64': 'm'}>
 CONFIG_DVB_M88RS2000                            policy<{'amd64': 'm', 'arm64': 'm', 'armhf': 'm', 'ppc64el': 'm', 'riscv64': 'm'}>
 CONFIG_DVB_MANTIS                               policy<{'amd64': 'm', 'arm64': 'm', 'armhf': 'm', 'ppc64el': 'm', 'riscv64': 'm'}>
-CONFIG_DVB_MAX_ADAPTERS                         policy<{'amd64': '8', 'arm64': '8', 'armhf': '8', 'ppc64el': '8', 'riscv64': '8'}>
+CONFIG_DVB_MAX_ADAPTERS                         policy<{'amd64': '32', 'arm64': '32', 'armhf': '32', 'ppc64el': '32', 'riscv64': '32'}>
 CONFIG_DVB_MB86A16                              policy<{'amd64': 'm', 'arm64': 'm', 'armhf': 'm', 'ppc64el': 'm', 'riscv64': 'm'}>
 CONFIG_DVB_MB86A20S                             policy<{'amd64': 'm', 'arm64': 'm', 'armhf': 'm', 'ppc64el': 'm', 'riscv64': 'm'}>
 CONFIG_DVB_MMAP                                 policy<{'amd64': 'n', 'arm64': 'n', 'armhf': 'n', 'ppc64el': 'n', 'riscv64': 'n', 's390x': '-'}>
-- 
2.43.0

