From 1eff8b001c5e557106649b9b875ae3d86f1c5459 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Thu, 21 Jan 2021 20:21:01 -0600
Subject: [PATCH 2/6] Changelog

---
 debian.master/changelog | 36 ++++++++++++++++++++++++++++++++++++
 1 file changed, 36 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index 3a2fb5f2e..a0253667c 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,39 @@
+linux (5.8.0-402101212021.0+mediatree+hauppauge) groovy; urgency=low
+
+  * Ubuntu-master kernel git clone
+    - 5.8.0-40.45 rev ecbe5fc7f6a719a099e7e777373e42786ce90a6b
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2020-10-10-10-10.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0004-si2168-different-default-that-windows-driver.patch
+    --- 0005-dw2102-check-initialization-replies.patch
+    --- 0006-dw2102-warn-on-failed-i2c-read-requests.patch
+    --- 0007-si2168-Include-b-4.0-11-when-checking-firmware-versi.patch
+    --- 0008-Fix-saa7164-REV2-firmware.patch
+    - 11.cx23885.patches
+    --- 0001-cx2885-fix-analog-video-routings.patch
+    --- 0002-cx23885-module-option-to-disable-analog-video.patch
+    --- 0003-cx23885-add-more-ryzen-pci-id-to-dma-override-list.patch
+    - 20.missing.stat.bits
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+    - 90.usb.quadhd.mxl692
+    --- 0001-mxl692-MaxLinear-692-ATSC-demod-tuner-driver.patch
+    --- 0002-em28xx-core-Fix-TS2-active-led.patch
+    --- 0003-em28xx-Add-support-for-Hauppauge-USB-QuadHD.patch
+
+ -- Fake User <hidden@email.co>  Thu, 21 Jan 2021 20:21:01 -0600
+
 linux (5.8.0-40.45) groovy; urgency=medium
 
   * Packaging resync (LP: #1786013)
-- 
2.27.0

