From 4b0623fd50e3bc9b60c2f6c5cf240fce0412cbf1 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Fri, 6 Nov 2020 13:21:30 -0600
Subject: [PATCH 2/6] Changelog

---
 debian.master/changelog | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index ae24e9d95..6139bf730 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,20 @@
+linux (5.8.0-252011061321.0+mediatree+hauppauge) groovy; urgency=low
+
+  * Ubuntu-master kernel git clone
+    - 5.8.0-25.26 rev 71cda6c56c6f808abaa4aaacc317eb24a9748a74
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2020-10-10-10-10.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+
+ -- Fake User <hidden@email.co>  Fri, 06 Nov 2020 13:21:30 -0600
+
 linux (5.8.0-25.26) groovy; urgency=medium
 
   * groovy/linux: 5.8.0-25.26 -proposed tracker (LP: #1899940)
-- 
2.27.0

