From 0ed414ad481d7adc702b5b9ca1e20591aa68b5e3 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Sun, 6 May 2018 07:08:09 -0500
Subject: [PATCH 2/6] Changelog

---
 debian.master/changelog | 56 +++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 56 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index 8202211e55d0..1c2eca0fd9e1 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,59 @@
+linux (4.13.0-39201805060708.0+mediatree+hauppauge) artful; urgency=low
+
+  * Ubuntu kernel git clone
+    - 4.13.0-39.44 rev 097ff12866439027cae08b9a6a2f73bc2731750f
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2018-04-30-04-20.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Add-new-media-tree-kernel-config.patch
+    - 0005-Mainline-Ubuntu-build-fixes.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 07.si2157.analog.tuning
+    --- 0001-si2157-Enable-tuner-status-flags.patch
+    --- 0002-si2157-Check-error-status-bit-on-cmd-execute.patch
+    --- 0003-si2157-Better-check-for-running-tuner-in-init.patch
+    --- 0004-si2157-Add-clock-and-pin-setup-for-si2141.patch
+    --- 0005-cx25840-Remove-offset-from-chip-specific-registers.patch
+    --- 0006-si2157-Add-analog-tuning-related-functions.patch
+    --- 0007-si2157-Briefly-wait-for-tuning-operation-to-complete.patch
+    --- 0008-cx23885-Add-analog-tuner-support-to-Hauppauge-QuadHD.patch
+    --- 0009-cx23885-Add-analog-frontend-to-1265_K4.patch
+    --- 0010-cx231xx-Add-i2c-device-analog-tuner-support.patch
+    --- 0011-cx23885-Add-i2c-device-analog-tuner-support.patch
+    - 08.cx23885.ryzen.dma.fixes
+    --- 0001-cx23885-Handle-additional-bufs-on-interrupt.patch
+    --- 0002-cx23885-Use-PCI-and-TS-masks-in-irq-functions.patch
+    --- 0003-cx23885-Expand-registers-printed-during-dma-tsport-r.patch
+    --- 0004-cx23885-Ryzen-DMA-related-RiSC-engine-stall-fixes.patch
+    --- 0005-cx23885-Add-some-missing-register-documentation.patch
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0005-si2168-different-default-that-windows-driver.patch
+    --- 0006-em28xx-bugfix-second-tuner-broken.patch
+    --- 0007-saa7164-Correct-seemingly-random-driver-debug.patch
+    --- 0008-cx231xx-Fix-recursive-dependency.patch
+    - 11.cx231xx.housework
+    --- 0001-cx231xx-Fix-several-incorrect-demod-addresses.patch
+    --- 0002-cx231xx-Use-board-profile-values-for-addresses.patch
+    --- 0003-cx231xx-Style-fix-for-struct-zero-init.patch
+    --- 0004-c231xx-disable-an-i2c-mux-adapter.patch
+    --- 0005-cx231xx-Switch-to-using-new-dvb-i2c-helpers.patch
+    --- 0006-cx231xx-Update-955Q-from-dvb-attach-to-i2c-device.patch
+    --- 0007-cx231xx-Remove-unnecessary-parameter-clear.patch
+    --- 0008-cx231xx-Remove-RC_CORE-dependency.patch
+    --- 0009-cx231xx-Add-I2C_MUX-dependency.patch
+    - 20.missing.stat.bits
+    --- 0001-si2157-add-on-demand-rf-strength-func.patch
+    --- 0002-lgdt3306a-Add-CNR-v5-stat.patch
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+
+ -- Fake User <hidden@email.co>  Sun, 06 May 2018 07:08:09 -0500
+
 linux (4.13.0-39.44) artful; urgency=medium
 
   * linux: 4.13.0-39.44 -proposed tracker (LP: #1761456)
-- 
2.14.1

