From e1feb3f1d187f51fe95ad00223546f5324765148 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Tue, 1 May 2018 21:09:10 -0500
Subject: [PATCH 2/6] Changelog

---
 debian.master/changelog | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index 8202211e55d0..9194fbf7f830 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,24 @@
+linux (4.13.0-39201805012109.0+mediatree+hauppauge) artful; urgency=low
+
+  * Ubuntu kernel git clone
+    - 4.13.0-39.44 rev 097ff12866439027cae08b9a6a2f73bc2731750f
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2018-04-30-04-20.patch
+    - 0002-Apply-build-fixes-to-media-tree.patch
+    - 0003-Add-new-media-tree-kernel-config.patch
+
+  * Ubuntu mainline build fixes
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+
+  * Packaging patches
+    - 0005-Packaging-updates.patch
+    - 0006-Changelog.patch
+
+  * Additional patches
+
+ -- Fake User <hidden@email.co>  Tue, 01 May 2018 21:09:10 -0500
+
 linux (4.13.0-39.44) artful; urgency=medium
 
   * linux: 4.13.0-39.44 -proposed tracker (LP: #1761456)
-- 
2.14.1

