From 4f24b0f798d1b538dcbaf90b3f7f4655a38d0e9e Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Sun, 11 Feb 2018 04:20:00 -0600
Subject: [PATCH 6/6] Changelog

---
 debian.master/changelog | 61 +++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 61 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index 8c81391ecb1d..0fd929146abe 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,64 @@
+linux (4.13.0-32201802110420.0+mediatree+hauppauge) artful; urgency=low
+
+  * Ubuntu kernel git clone
+    - 4.13.0-32.35 rev b32038eb34ee42fd8056f99f88652270f6667996
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2018-02-08-04-20.patch
+    - 0002-Apply-build-fixes-to-media-tree.patch
+    - 0003-Add-new-media-tree-kernel-config.patch
+
+  * Ubuntu mainline build fixes
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+
+  * Packaging patches
+    - 0005-Packaging-updates.patch
+    - 0006-Changelog.patch
+
+  * Additional patches
+    - 0001-em28xx-Hauppauge-DualHD-second-tuner-functionality.patch
+    - 0002-em28xx-Bulk-transfer-implementation-fix.patch
+    - 0003-em28xx-USB-bulk-packet-size-fix.patch
+    - 0004-em28xx-Increase-max-em28xx-boards-to-max-dvb-adapter.patch
+    - 0005-em28xx-Add-Hauppauge-SoloHD-DualHD-bulk-models.patch
+    - 0006-em28xx-Enable-Hauppauge-SoloHD-rebranded-292e-SE.patch
+    - 0007-lgdt3306a-Set-fe-ops.release-to-NULL-if-probed.patch
+    - 0008-lgdt3306a-QAM-streaming-improvement.patch
+    - 0009-lgdt3306a-Add-QAM-AUTO-support.patch
+    - 0010-si2168-Add-spectrum-inversion-property.patch
+    - 0011-em28xx-Enable-inversion-for-Solo-Dual-HD-DVB-models.patch
+    - 0012-cx231xx-Add-support-for-Hauppauge-HVR-935C.patch
+    - 0013-cx231xx-Add-support-for-Hauppauge-HVR-975.patch
+    - 0014-cx231xx-Add-second-frontend-option.patch
+    - 0015-cx231xx-Add-second-i2c-demod-client.patch
+    - 0016-si2168-Add-ts-bus-control-turn-off-bus-on-sleep.patch
+    - 0017-si2168-Announce-frontend-creation-failure.patch
+    - 0018-lgdt3306a-Announce-successful-creation.patch
+    - 0019-cx231xx-Add-second-i2c-demod-to-Hauppauge-975.patch
+    - 0020-cx23885-Enable-new-Hauppauge-PCIe-ImpactVCBe-variant.patch
+    - 0021-cx23885-Add-support-for-Hauppauge-PCIe-HVR1265-K4.patch
+    - 0022-cx23885-Add-support-for-Hauppauge-PCIe-Starburst2.patch
+    - 0023-cx23885-Add-support-for-new-Hauppauge-QuadHD-885.patch
+    - 0024-cx23885-Fix-gpio-on-Hauppauge-QuadHD-PCIe-cards.patch
+    - 0025-cx25840-Use-subdev-host-data-for-PLL-override.patch
+    - 0026-cx23885-change-887-888-default-to-888.patch
+    - 0027-cx23885-Set-subdev-host-data-to-clk_freq-pointer.patch
+    - 0028-cx23885-Override-888-ImpactVCBe-crystal-frequency.patch
+    - 0029-si2157-Enable-tuner-status-flags.patch
+    - 0030-si2157-Check-error-status-bit-on-cmd-execute.patch
+    - 0031-si2157-Better-check-for-running-tuner-in-init.patch
+    - 0032-si2157-Add-clock-and-pin-setup-for-si2141.patch
+    - 0033-cx25840-Remove-offset-from-chip-specific-registers.patch
+    - 0034-si2157-Add-analog-tuning-related-functions.patch
+    - 0035-si2157-Briefly-wait-for-tuning-operation-to-complete.patch
+    - 0036-cx23885-Add-analog-tuner-support-to-Hauppauge-QuadHD.patch
+    - 0037-cx23885-Add-analog-frontend-to-1265_K4.patch
+    - 0038-cx231xx-Add-i2c-device-analog-support.patch
+    - 0039-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    - 0040-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+
+ -- Fake User <hidden@email.co>  Sun, 11 Feb 2018 04:20:00 -0600
+
 linux (4.13.0-32.35) artful; urgency=low
 
   * CVE-2017-5715 // CVE-2017-5753
-- 
2.14.1

