From 499ce8d5d5dee8ffc55b9feb9f45794be1dba153 Mon Sep 17 00:00:00 2001
From: Brad Love <brad@nextdimension.cc>
Date: Thu, 26 Oct 2017 11:11:11 +0000
Subject: [PATCH 6/6] Changelog

---
 debian.master/changelog | 45 +++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 45 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index aaae08d9a9c6..9b8029faf40e 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,48 @@
+linux (4.13.0-16201710261111.0+mediatree+hauppauge) artful; urgency=low
+
+  * Ubuntu kernel git clone
+    - 4.13.0-16.19 rev 5cd9ee4eae93973818b75acb1cd84cbb273af43f
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2017-10-05-20-10.patch
+    - 0002-Apply-build-fixes-to-media-tree.patch
+    - 0003-Add-new-media-tree-kernel-config.patch
+
+  * Ubuntu mainline build fixes
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+
+  * Packaging patches
+    - 0005-Packaging-updates.patch
+    - 0006-Changelog.patch
+
+  * Additional patches
+    - 0001-em28xx-Hauppauge-DualHD-DVB-ATSC-second-tuner-suppor.patch
+    - 0002-em28xx-bulk-transfer-implementation-fix.patch
+    - 0003-em28xx-usb-packet-size-tweaks.patch
+    - 0004-lgdt3306a-add-QAM-AUTO-support.patch
+    - 0005-em28xx-Add-Hauppauge-SoloHD-DualHD-bulk-models-suppo.patch
+    - 0006-cx23885-Enable-new-Hauppauge-PCIe-ImpactVCB-variant.patch
+    - 0007-cx23885-set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    - 0008-si2157-check-error-status-bit-on-cmd-execute.patch
+    - 0009-si2157-briefly-wait-for-tuning-operation-to-complete.patch
+    - 0010-Si2157-better-check-for-running-tuner-in-init-Reset-.patch
+    - 0011-si2157-add-analog-tuner-support-and-dvb-frontend.patch
+    - 0012-cx23xxx-Ensure-i2c-platform-data-object-lifetime.patch
+    - 0013-cx23885-add-support-for-Hauppauge-HVR1265-K4.patch
+    - 0014-cx25840-remove-offset-from-chip-specific-registers-A.patch
+    - 0015-cx28335-Add-support-for-Hauppauge-Starburst-2-PCIe-D.patch
+    - 0016-cx23885-Add-support-for-new-Hauppauge-QuadHD-885.patch
+    - 0017-cx23885-Add-analog-tuner-support-to-Hauppauge-QuadHD.patch
+    - 0018-cx23885-fix-gpio-on-Hauppauge-QuadHD-888-PCIe-cards.patch
+    - 0019-lgdt3306a-QAM-streaming-improvement-Required-for-sta.patch
+    - 0020-si2168-spectral-inversion-property-Hauppauge-DualHD-.patch
+    - 0021-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    - 0022-em28xx-Enable-Hauppauge-SoloHD-rebranded-292e-SE.patch
+    - 0023-em28xx-increase-max-em28xx-boards-to-max-dvb-adapter.patch
+    - 0024-v4l2-core-Reverse-758d90e16-kvmalloc-patch.patch
+
+ -- Fake User <hidden@email.co>  Thu, 26 Oct 2017 11:11:11 +0000
+
 linux (4.13.0-16.19) artful; urgency=low
 
   * 20170817 - ISO hangs on boot on qemu with splash screen enabled and qxl
-- 
2.14.1

