From d29ac1ef8ee185d64ac699a84ce3b39af2af7cc4 Mon Sep 17 00:00:00 2001
From: Brad Love <brad@nextdimension.cc>
Date: Mon, 21 Dec 2020 15:18:11 -0600
Subject: [PATCH 3/3] cx23885: add more ryzen pci id to dma override list

---
 drivers/media/pci/cx23885/cx23885-core.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/media/pci/cx23885/cx23885-core.c b/drivers/media/pci/cx23885/cx23885-core.c
index 2cb8a0263ec1..d4669606bda8 100644
--- a/drivers/media/pci/cx23885/cx23885-core.c
+++ b/drivers/media/pci/cx23885/cx23885-core.c
@@ -2085,6 +2085,8 @@ static struct {
 	 * 0x1451 is PCI ID for the IOMMU found on Ryzen
 	 */
 	{ PCI_VENDOR_ID_AMD, 0x1451 },
+	{ PCI_VENDOR_ID_AMD, 0x1481 },
+	{ PCI_VENDOR_ID_AMD, 0x1419 },
 };
 
 static bool cx23885_does_need_dma_reset(void)
-- 
2.28.0

