From 7497ce02051f686ea12358534a38597d49b0ecce Mon Sep 17 00:00:00 2001
From: Bradford Love <hidden@email.co>
Date: Tue, 28 Feb 2023 12:58:38 -0600
Subject: [PATCH 2/6] Changelog

---
 debian.raspi/changelog | 31 +++++++++++++++++++++++++++++++
 1 file changed, 31 insertions(+)

diff --git a/debian.raspi/changelog b/debian.raspi/changelog
index abf3a7605..793fd3c91 100644
--- a/debian.raspi/changelog
+++ b/debian.raspi/changelog
@@ -1,3 +1,34 @@
+linux-raspi (5.4.0-1312302280420.0+mediatree+hauppauge~raspi) focal; urgency=low
+
+  * Ubuntu-raspi kernel git clone
+    - 5.4.0-131.147 rev a2e7b009997aa16e93c10b4dbae101811a50935e
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2022-06-06-06-00.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0004-si2168-different-default-that-windows-driver.patch
+    --- 0005-dw2102-check-initialization-replies.patch
+    --- 0006-dw2102-warn-on-failed-i2c-read-requests.patch
+    --- 0007-si2168-Include-b-4.0-11-when-checking-firmware-versi.patch
+    --- 0008-Fix-saa7164-REV2-firmware.patch
+    --- 0009-Revert.2c98b8a34.patch
+    - 11.cx23885.patches
+    --- 0002-cx23885-module-option-to-disable-analog-video.patch
+    - 20.missing.stat.bits
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+
+ -- Fake User <hidden@email.co>  Tue, 28 Feb 2023 04:20:00 -0600
+
 linux-raspi (5.4.0-1073.84) focal; urgency=medium
 
   [ Ubuntu: 5.4.0-131.147 ]
-- 
2.34.1

