From 1803e11096a9df00c63060f6e616b7e931f726f7 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Tue, 20 Jun 2017 06:20:46 -0500
Subject: [PATCH 6/6] Changelog

---
 debian.master/changelog | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index 83c9fedcc587..ebf7d4c1f3ce 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,29 @@
+linux (4.10.0-24201706200620.0+mediatree+hauppauge) zesty; urgency=low
+
+  * Ubuntu kernel git clone
+    - 4.10.0-24.28 rev 11067176f5d6bcc44dc73ab355a30c6838c597b0
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2017-06-20-04-20.patch
+    - 0002-Apply-build-fixes-to-media-tree.patch
+    - 0003-Add-new-media-tree-kernel-config.patch
+
+  * Ubuntu mainline build fixes
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+
+  * Packaging patches
+    - 0005-Packaging-updates.patch
+    - 0006-Changelog.patch
+
+  * Additional patches
+    - 0001-em28xx-Hauppauge-DualHD-DVB-ATSC-second-tuner-suppor.patch
+    - 0002-em28xx-bulk-transfer-implementation-fix.patch
+    - 0003-em28xx-usb-packet-size-tweaks.patch
+    - 0004-lgdt3306a-add-QAM-AUTO-support.patch
+    - 0005-em28xx-Add-Hauppauge-SoloHD-DualHD-bulk-models-suppo.patch
+
+ -- Fake User <hidden@email.co>  Tue, 20 Jun 2017 06:20:46 -0500
+
 linux (4.10.0-24.28) zesty; urgency=low
 
   * CVE-2017-1000364
-- 
2.11.0

