From 8b9cca3af12e36c234c1e32c9ada26ec144482af Mon Sep 17 00:00:00 2001
From: root <hidden@email.co>
Date: Tue, 10 Oct 2017 18:55:20 +0000
Subject: [PATCH 3/6] Add 'new' media tree kernel config

---
 debian.master/config/config.common.ubuntu | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/debian.master/config/config.common.ubuntu b/debian.master/config/config.common.ubuntu
index c5601c189c6e..ea299f95b3ad 100644
--- a/debian.master/config/config.common.ubuntu
+++ b/debian.master/config/config.common.ubuntu
@@ -10037,3 +10037,9 @@ CONFIG_ZSMALLOC=y
 CONFIG_ZSWAP=y
 CONFIG_ZX2967_PM_DOMAINS=y
 CONFIG_ZX_TDM=m
+### LINUX MEDIA TREE EXTRA OPTIONS ###
+CONFIG_IR_GPIO_TX=m
+CONFIG_IR_PWM_TX=m
+# CONFIG_DVB_DDBRIDGE_MSIENABLE is not set
+# CONFIG_SDR_PLATFORM_DRIVERS is not set
+
-- 
2.11.0

