From 0d901e21450ddb481d1a9aad88607780b2102f4a Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Mon, 13 Feb 2023 14:25:49 -0600
Subject: [PATCH 1/6] Linuxtv.org media tree sync - 2022-06-06-06-00

---
 debian/empty | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 debian/empty

diff --git a/debian/empty b/debian/empty
new file mode 100644
index 000000000..e1775d5f3
--- /dev/null
+++ b/debian/empty
@@ -0,0 +1,2 @@
+Placeholder
+
-- 
2.37.2

