From 7e8a4c557cffe01ad5c0c13e1c00458550140d8d Mon Sep 17 00:00:00 2001
From: Michael Bunk <micha@freedict.org>
Date: Sat, 4 Jan 2020 20:08:22 +0100
Subject: [PATCH 3/6] dw2102: Log probed demodulator type and address

Signed-off-by: Michael Bunk <micha@freedict.org>
---
 drivers/media/usb/dvb-usb/dw2102.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/media/usb/dvb-usb/dw2102.c b/drivers/media/usb/dvb-usb/dw2102.c
index b960abd00..472448247 100644
--- a/drivers/media/usb/dvb-usb/dw2102.c
+++ b/drivers/media/usb/dvb-usb/dw2102.c
@@ -1589,6 +1589,9 @@ static int tt_s2_4600_frontend_attach(struct dvb_usb_adapter *adap)
 	strscpy(board_info.type, "m88ds3103", I2C_NAME_SIZE);
 	board_info.addr = 0x68;
 	board_info.platform_data = &m88ds3103_pdata;
+
+	info("%s: attaching demodulator of type %s at i2c address 0x%x", __func__, board_info.type, board_info.addr);
+
 	request_module("m88ds3103");
 	client = i2c_new_device(&d->i2c_adap, &board_info);
 	if (client == NULL || client->dev.driver == NULL)
-- 
2.20.1

