From ffb05add180bf8f22167d805494a4a51be8d7da9 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Sat, 11 Jul 2020 07:11:00 -0500
Subject: [PATCH 2/6] Changelog

---
 debian.hwe/changelog | 30 ++++++++++++++++++++++++++++++
 1 file changed, 30 insertions(+)

diff --git a/debian.hwe/changelog b/debian.hwe/changelog
index 37c652e..26e168c 100644
--- a/debian.hwe/changelog
+++ b/debian.hwe/changelog
@@ -1,3 +1,33 @@
+linux-hwe (4.15.0-110202007110711.0+mediatree+hauppauge~hwe) xenial; urgency=low
+
+  * Ubuntu-hwe kernel git clone
+    - 4.15.0-110.111 rev 2d5b3a4a38cc87b3add231aea9e9e8146e4a24ff
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2020-07-07-07-07.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0004-si2168-different-default-that-windows-driver.patch
+    --- 0005-dw2102-check-initialization-replies.patch
+    --- 0006-dw2102-warn-on-failed-i2c-read-requests.patch
+    - 20.missing.stat.bits
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+    - 90.usb.quadhd.xml692
+    --- 0001-mxl69x-MaxLinear-692-ATSC-demod-tuner-driver.patch
+    --- 0002-em28xx-core-Fix-TS2-active-led.patch
+    --- 0003-em28xx-Add-support-for-Hauppauge-USB-QuadHD.patch
+
+ -- Fake User <hidden@email.co>  Sat, 11 Jul 2020 07:11:00 -0500
+
 linux-hwe (4.15.0-110.111~16.04.1) xenial; urgency=medium
 
   * xenial/linux-hwe: 4.15.0-110.111~16.04.1 -proposed tracker (LP: #1885813)
-- 
2.7.4

