From 6928102a8d80827b51658d9a640427fe8c0a6d7c Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Thu, 9 Sep 2021 16:50:19 -0500
Subject: [PATCH 2/6] Changelog

---
 debian.hwe/changelog | 30 ++++++++++++++++++++++++++++++
 1 file changed, 30 insertions(+)

diff --git a/debian.hwe/changelog b/debian.hwe/changelog
index e6246d1..eb38a35 100644
--- a/debian.hwe/changelog
+++ b/debian.hwe/changelog
@@ -1,3 +1,33 @@
+linux-hwe (4.15.0-142202109090909.0+mediatree+hauppauge~hwe) xenial; urgency=low
+
+  * Ubuntu-hwe kernel git clone
+    - 4.15.0-142.146 rev 523f0632baab29fe0e679315fe693ee10217deda
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2021-09-09-09-09.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0004-si2168-different-default-that-windows-driver.patch
+    --- 0005-dw2102-check-initialization-replies.patch
+    --- 0006-dw2102-warn-on-failed-i2c-read-requests.patch
+    --- 0007-si2168-Include-b-4.0-11-when-checking-firmware-versi.patch
+    --- 0008-Fix-saa7164-REV2-firmware.patch
+    - 11.cx23885.patches
+    --- 0002-cx23885-module-option-to-disable-analog-video.patch
+    - 20.missing.stat.bits
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+
+ -- Fake User <hidden@email.co>  Thu, 09 Sep 2021 09:09:09 -0500
+
 linux-hwe (4.15.0-142.146~16.04.1) xenial; urgency=medium
 
   * P8 node entei unable to boot with 4.15.0-141.145~16.04.1 (LP: #1922997)
-- 
2.7.4

