From 9ed3b5751daea44770194767b09f4b6bbf5e47f0 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Wed, 2 Oct 2019 20:01:19 -0500
Subject: [PATCH 1/5] Changelog

---
 debian.hwe/changelog | 47 ++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 47 insertions(+)

diff --git a/debian.hwe/changelog b/debian.hwe/changelog
index d926b2ffd281..752b387876ba 100644
--- a/debian.hwe/changelog
+++ b/debian.hwe/changelog
@@ -1,3 +1,50 @@
+linux-hwe (5.0.0-32201910022001.1+mediatree+hauppauge~hwe) bionic; urgency=low
+
+  * Ubuntu-hwe kernel git clone
+    - 5.0.0-32.34 rev b9890b08d6f9ca3d3d50b91206dc2d3ff6411377
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 01.HVR19xx
+    --- 0001-si2157-add-detection-of-si2177-tuner.patch
+    --- 0002-pvrusb2-Add-multiple-dvb-frontend-support.patch
+    --- 0003-pvrusb2-Add-i2c-client-demod-tuner-support.patch
+    --- 0004-pvrusb2-Add-Hauppauge-HVR1955-1975-devices.patch
+    - 07.si2157.analog.tuning
+    --- 0001-si2157-Enable-tuner-status-flags.patch
+    --- 0002-si2157-Check-error-status-bit-on-cmd-execute.patch
+    --- 0003-si2157-Better-check-for-running-tuner-in-init.patch
+    --- 0004-si2157-Add-clock-and-pin-setup-for-si2141.patch
+    --- 0005-cx25840-Remove-offset-from-chip-specific-registers.patch
+    --- 0006-si2157-Add-analog-tuning-related-functions.patch
+    --- 0007-si2157-Briefly-wait-for-tuning-operation-to-complete.patch
+    --- 0008-cx23885-Add-analog-tuner-support-to-Hauppauge-QuadHD.patch
+    --- 0009-cx23885-Add-analog-frontend-to-1265_K4.patch
+    --- 0010-cx231xx-Add-i2c-device-analog-tuner-support.patch
+    --- 0011-cx23885-Add-i2c-device-analog-tuner-support.patch
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0005-si2168-different-default-that-windows-driver.patch
+    --- 0020-si2168-add-frquency-data-to-frontend-info.patch
+    --- 0030-Ignore-first-cold-command-error-for-si2141-in-_init.patch
+    - 20.missing.stat.bits
+    --- 0001-si2157-add-on-demand-rf-strength-func.patch
+    --- 0002-lgdt3306a-Add-CNR-v5-stat.patch
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+    - 99.montage.3103b
+    --- 0001-m88ds3103-Add-Montage-3103b-support.patch
+    --- 0002-em28xx-Add-Hauppauge-461e-revision-2.patch
+
+ -- Fake User <hidden@email.co>  Wed, 2 Oct 2019 20:01:19 -0500
+
 linux-hwe (5.0.0-32.34~18.04.1) bionic; urgency=medium
 
   * bionic/linux-hwe: 5.0.0-32.34~18.04.1 -proposed tracker (LP: #1846092)
-- 
2.17.1

