From 120cbeb389b45069876e09501a576d98744bceeb Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Wed, 28 Jan 2026 01:28:26 -0600
Subject: [PATCH 2/6] Changelog

---
 debian.hwe-5.4/changelog | 47 ++++++++++++++++++++++++++++++++++++++++
 1 file changed, 47 insertions(+)

diff --git a/debian.hwe-5.4/changelog b/debian.hwe-5.4/changelog
index 3309f8812009..d0d162d08584 100644
--- a/debian.hwe-5.4/changelog
+++ b/debian.hwe-5.4/changelog
@@ -1,3 +1,50 @@
+linux-hwe (5.4.0-1522601280128.0+mediatree+hauppauge~hwe) bionic; urgency=low
+
+  * Ubuntu hwe-5.4 kernel git clone
+    - 5.4.0-152.169 rev 3d5133ff256678ababe7a98c3519043d7f656314
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2023-05-24-04-20.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0004-si2168-different-default-that-windows-driver.patch
+    --- 0005-dw2102-check-initialization-replies.patch
+    --- 0006-dw2102-warn-on-failed-i2c-read-requests.patch
+    --- 0007-si2168-Include-b-4.0-11-when-checking-firmware-versi.patch
+    --- 0008-Fix-saa7164-REV2-firmware.patch
+    --- 0009-cx231xx-Fix-AGC-configuration-to-defaults-from-windo.patch
+    --- 0010-au0828-Add-dqbuf-function-to-check-for-and-fix-green.patch
+    --- 0011-xc5000-rf-strength-function-is-required-to-get-analo.patch
+    --- 0012-cx25840-Fix-NTSC-J-PAL-N-and-SECAM-standards.patch
+    --- 0013-si2157-Fix-SECAM-L-PAL-N-and-PAL-Nc.patch
+    --- 0014-au0828-Add-new-Hauppauge-HVR1265-and-ImpactVCB-e.patch
+    --- 0015-em28xx-Add-DualHD-variants.patch
+    - 11.cx23885.patches
+    --- 0002-cx23885-module-option-to-disable-analog-video.patch
+    - 20.missing.stat.bits
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+    - 25.em2828X
+    --- 0001-Add-support-for-EM2828X.patch
+    --- 0002-em28xx-Add-EM2828X-based-Hauppauge-USB-Live2.patch
+    - 27.em2828x.based.9x5
+    --- 0001-em28xx-Include-support-for-second-i2c-client-demod.patch
+    --- 0002-si2157-Add-analog-TV-support-for-si2177.patch
+    --- 0003-em28xx-Add-Hauppauge-em2828X-based-9x5-revisions.patch
+    - 30.Montage.3103c.demod
+    --- 0001-m88ds3103-Add-Montage-3103c-demod-support.patch
+    --- 0002-em28xx-Add-Hauppauge-461e-v3.patch
+
+ -- Fake User <hidden@email.co>  Wed, 28 Jan 2026 01:28:26 -0600
+
 linux-hwe-5.4 (5.4.0-152.169~18.04.1) bionic; urgency=medium
 
   * bionic/linux-hwe-5.4: 5.4.0-152.169~18.04.1 -proposed tracker (LP: #2023103)
-- 
2.17.1

