From a19950e01a772c14fa623a9782f072e5daddbbc9 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Thu, 12 May 2022 16:20:00 -0500
Subject: [PATCH 2/6] Changelog

---
 debian.hwe-5.4/changelog | 31 +++++++++++++++++++++++++++++++
 1 file changed, 31 insertions(+)

diff --git a/debian.hwe-5.4/changelog b/debian.hwe-5.4/changelog
index 368c63d269a1..6d520cc945d9 100644
--- a/debian.hwe-5.4/changelog
+++ b/debian.hwe-5.4/changelog
@@ -1,3 +1,34 @@
+linux-hwe (5.4.0-1122205121620.0+mediatree+hauppauge~hwe) bionic; urgency=low
+
+  * Ubuntu-hwe-5.4 kernel git clone
+    - 5.4.0-112.126 rev fadd782e0a1de9d9151cc8b1548aa70b89e66867
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2022-02-02-20-22.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0004-si2168-different-default-that-windows-driver.patch
+    --- 0005-dw2102-check-initialization-replies.patch
+    --- 0006-dw2102-warn-on-failed-i2c-read-requests.patch
+    --- 0007-si2168-Include-b-4.0-11-when-checking-firmware-versi.patch
+    --- 0008-Fix-saa7164-REV2-firmware.patch
+    --- 0009-Revert.2c98b8a34.patch
+    - 11.cx23885.patches
+    --- 0002-cx23885-module-option-to-disable-analog-video.patch
+    - 20.missing.stat.bits
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+
+ -- Fake User <hidden@email.co>  Thu, 12 May 2022 16:20:00 -0500
+
 linux-hwe-5.4 (5.4.0-112.126~18.04.1) bionic; urgency=medium
 
   * bionic/linux-hwe-5.4: 5.4.0-112.126~18.04.1 -proposed tracker (LP: #1971838)
-- 
2.17.1

