From ab439f046c6268fdb1df6b875d3e7107d4094083 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Mon, 30 Apr 2018 13:06:11 -0500
Subject: [PATCH 5/5] Changelog

---
 debian.master/changelog | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index fe5e6e597835..42d27b31fc6a 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,24 @@
+linux (4.15.0-20201804301306.0+mediatree+hauppauge) bionic; urgency=low
+
+  * Ubuntu kernel git clone
+    - 4.15.0-20.21 rev d4174baeefbd0a9634291145600b90e13163f27d
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2018-04-30-04-20.patch
+    - 0002-Apply-build-fixes-to-media-tree.patch
+    - 0003-Add-new-media-tree-kernel-config.patch
+
+  * Ubuntu mainline build fixes
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+
+  * Packaging patches
+    - 0005-Packaging-updates.patch
+    - 0006-Changelog.patch
+
+  * Additional patches
+
+ -- Fake User <hidden@email.co>  Mon, 30 Apr 2018 13:06:11 -0500
+
 linux (4.15.0-20.21) bionic; urgency=medium
 
   * linux: 4.15.0-20.21 -proposed tracker (LP: #1766452)
-- 
2.15.1

