From 6b138da5642a7204b700d59b561c81b7c82e06cc Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Tue, 20 Oct 2020 10:20:10 -0500
Subject: [PATCH 2/6] Changelog

---
 debian.master/changelog | 31 +++++++++++++++++++++++++++++++
 1 file changed, 31 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index ab4a95ce628d..98be901239a0 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,34 @@
+linux (4.15.0-121202010201020.0+mediatree+hauppauge) bionic; urgency=low
+
+  * Ubuntu-master kernel git clone
+    - 4.15.0-121.123 rev 59c4ac7cff7e0267aed27bde64e8854e5a35cec6
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2020-07-07-07-07.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0004-si2168-different-default-that-windows-driver.patch
+    --- 0005-dw2102-check-initialization-replies.patch
+    --- 0006-dw2102-warn-on-failed-i2c-read-requests.patch
+    --- 0007-si2168-Include-b-4.0-11-when-checking-firmware-versi.patch
+    - 20.missing.stat.bits
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+    - 90.usb.quadhd.xml692
+    --- 0001-mxl69x-MaxLinear-692-ATSC-demod-tuner-driver.patch
+    --- 0002-em28xx-core-Fix-TS2-active-led.patch
+    --- 0003-em28xx-Add-support-for-Hauppauge-USB-QuadHD.patch
+
+ -- Fake User <hidden@email.co>  Tue, 20 Oct 2020 10:20:10 -0500
+
 linux (4.15.0-121.123) bionic; urgency=medium
 
   * Packaging resync (LP: #1786013)
-- 
2.17.1

