From 63a17761c11fa60cbdfa66390690d44f2c8e89f9 Mon Sep 17 00:00:00 2001
From: Bradford Love <hidden@email.co>
Date: Wed, 20 Apr 2022 14:18:30 -0500
Subject: [PATCH 1/6] Linuxtv.org media tree sync - 2022-04-20-04-20

---
 debian/empty | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 debian/empty

diff --git a/debian/empty b/debian/empty
new file mode 100644
index 000000000..e1775d5f3
--- /dev/null
+++ b/debian/empty
@@ -0,0 +1,2 @@
+Placeholder
+
-- 
2.34.1

