From 738f8d0f58528e50942618e918fbe5e2bd3859ac Mon Sep 17 00:00:00 2001
From: Bradford Love <hidden@email.co>
Date: Wed, 10 Aug 2022 12:04:30 -0500
Subject: [PATCH 5/6] Add new media tree kernel config

---
 debian.master/config/config.common.ubuntu | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/debian.master/config/config.common.ubuntu b/debian.master/config/config.common.ubuntu
index 9f487b2b8..eba685711 100644
--- a/debian.master/config/config.common.ubuntu
+++ b/debian.master/config/config.common.ubuntu
@@ -12821,3 +12821,18 @@ CONFIG_ZYNQMP_FIRMWARE=y
 CONFIG_ZYNQMP_IPI_MBOX=y
 CONFIG_ZYNQMP_PM_DOMAINS=y
 CONFIG_ZYNQMP_POWER=y
+### LINUX MEDIA TREE EXTRA OPTIONS ###
+CONFIG_MEDIA_PLATFORM_DRIVERS=y
+CONFIG_VIDEO_HI846=m
+CONFIG_VIDEO_HI847=m
+CONFIG_VIDEO_OG01A1B=m
+CONFIG_VIDEO_OV08D10=m
+CONFIG_VIDEO_OV13B10=m
+CONFIG_VIDEO_OV5693=m
+CONFIG_VIDEO_ZORAN_DC30=y
+CONFIG_VIDEO_ZORAN_ZR36060=y
+CONFIG_VIDEO_ZORAN_BUZ=y
+CONFIG_VIDEO_ZORAN_DC10=y
+CONFIG_VIDEO_ZORAN_LML33=y
+CONFIG_VIDEO_ZORAN_LML33R10=y
+CONFIG_VIDEO_ZORAN_AVS6EYES=y
-- 
2.34.1

