From e0f12cbf1c7ac5aeaf2bea097e4d2c8fbea093a5 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Tue, 28 May 2024 04:20:00 -0500
Subject: [PATCH 2/6] Changelog

---
 debian.master/changelog | 39 +++++++++++++++++++++++++++++++++++++++
 1 file changed, 39 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index c247c77ba..a896af545 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,42 @@
+linux (5.4.0-1862405280420.0+mediatree+hauppauge) focal; urgency=low
+
+  * Ubuntu master-next kernel git clone
+    - 5.4.0-186.206 rev 9bbfaa01721018bfe680a9c39d82c3f786efdf09
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2023-05-24-04-20.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0004-si2168-different-default-that-windows-driver.patch
+    --- 0005-dw2102-check-initialization-replies.patch
+    --- 0006-dw2102-warn-on-failed-i2c-read-requests.patch
+    --- 0007-si2168-Include-b-4.0-11-when-checking-firmware-versi.patch
+    --- 0008-Fix-saa7164-REV2-firmware.patch
+    --- 0011-cx231xx-Fix-AGC-configuration-to-defaults-from-windo.patch
+    --- 0012-au0828-Add-dqbuf-function-to-check-for-and-fix-green.patch
+    --- 0013-xc5000-rf-strength-function-is-required-to-get-analo.patch
+    --- 0014-cx25840-Fix-NTSC-J-PAL-N-and-SECAM-standards.patch
+    --- 0015-si2157-Fix-SECAM-L-PAL-N-and-PAL-Nc.patch
+    --- 0016-au0828-Add-new-Hauppauge-HVR1265-and-ImpactVCB-e.patch
+    --- 0017-em28xx-Add-DualHD-variants.patch
+    --- 0018-em28xx-Add-support-for-EM2828X.patch
+    --- 0019-em28xx-Add-support-for-em2828X-based-USB-Live2.patch
+    - 11.cx23885.patches
+    --- 0002-cx23885-module-option-to-disable-analog-video.patch
+    - 20.missing.stat.bits
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+
+ -- Fake User <hidden@email.co>  Tue, 28 May 2024 04:20:00 -0500
+
 linux (5.4.0-186.206) focal; urgency=medium
 
   * focal/linux: 5.4.0-186.206 -proposed tracker (LP: #2063812)
-- 
2.25.1

