From 6de1c88a8f990c1385e7c51c33ddbb64da1cc563 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Mon, 1 Nov 2021 13:24:22 -0500
Subject: [PATCH 2/6] Changelog

---
 debian.master/changelog | 30 ++++++++++++++++++++++++++++++
 1 file changed, 30 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index 7622f4960..36576b9f5 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,33 @@
+linux (5.13.0-192110311031.0+mediatree+hauppauge) impish; urgency=low
+
+  * Ubuntu-master kernel git clone
+    - 5.13.0-19.19 rev 86d5f4d4ce66d96657de67b735dacb25b8ab8a1b
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2021-09-09-09-09.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0004-si2168-different-default-that-windows-driver.patch
+    --- 0005-dw2102-check-initialization-replies.patch
+    --- 0006-dw2102-warn-on-failed-i2c-read-requests.patch
+    --- 0007-si2168-Include-b-4.0-11-when-checking-firmware-versi.patch
+    --- 0008-Fix-saa7164-REV2-firmware.patch
+    - 11.cx23885.patches
+    --- 0002-cx23885-module-option-to-disable-analog-video.patch
+    - 20.missing.stat.bits
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+
+ -- Fake User <hidden@email.co>  Sun, 31 Oct 2021 10:31:00 -0500
+
 linux (5.13.0-19.19) impish; urgency=medium
 
   * impish/linux: 5.13.0-19.19 -proposed tracker (LP: #1946337)
-- 
2.32.0

