From 242c5faaf16d467ff3da5a3fb6eaa4568d6725d3 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Mon, 12 Aug 2019 16:20:00 -0500
Subject: [PATCH 2/6] Changelog

---
 debian.master/changelog | 50 +++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 50 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index 67b53d3..3acd2cd 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,53 @@
+linux (4.4.0-157201908121620.0+mediatree+hauppauge) xenial; urgency=low
+
+  * Ubuntu-master kernel git clone
+    - 4.4.0-157.185 rev c908e95069c15c7455a2c1191b791dd1f2ef97e1
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2019-01-09-04-20.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 01.HVR19xx
+    --- 0001-si2157-add-detection-of-si2177-tuner.patch
+    --- 0002-pvrusb2-Add-multiple-dvb-frontend-support.patch
+    --- 0003-pvrusb2-Add-i2c-client-demod-tuner-support.patch
+    --- 0004-pvrusb2-Add-Hauppauge-HVR1955-1975-devices.patch
+    - 07.si2157.analog.tuning
+    --- 0001-si2157-Enable-tuner-status-flags.patch
+    --- 0002-si2157-Check-error-status-bit-on-cmd-execute.patch
+    --- 0003-si2157-Better-check-for-running-tuner-in-init.patch
+    --- 0004-si2157-Add-clock-and-pin-setup-for-si2141.patch
+    --- 0005-cx25840-Remove-offset-from-chip-specific-registers.patch
+    --- 0006-si2157-Add-analog-tuning-related-functions.patch
+    --- 0007-si2157-Briefly-wait-for-tuning-operation-to-complete.patch
+    --- 0008-cx23885-Add-analog-tuner-support-to-Hauppauge-QuadHD.patch
+    --- 0009-cx23885-Add-analog-frontend-to-1265_K4.patch
+    --- 0010-cx231xx-Add-i2c-device-analog-tuner-support.patch
+    --- 0011-cx23885-Add-i2c-device-analog-tuner-support.patch
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0005-si2168-different-default-that-windows-driver.patch
+    --- 0020-si2168-add-frquency-data-to-frontend-info.patch
+    - 20.missing.stat.bits
+    --- 0001-si2157-add-on-demand-rf-strength-func.patch
+    --- 0002-lgdt3306a-Add-CNR-v5-stat.patch
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+    - 40.cx231xx.media.controller.fix
+    --- 0001-cx231xx-video-Set-media-controller-taint-for-analog-.patch
+    --- 0002-cx25840-core-Set-media-controller-taint-for-pads.patch
+    - 99.montage.3103b
+    --- 0001-m88ds3103-Add-Montage-3103b-support.patch
+    --- 0002-em28xx-Add-Hauppauge-461e-revision-2.patch
+
+ -- Fake User <hidden@email.co>  Mon, 12 Aug 2019 16:20:00 -0500
+
 linux (4.4.0-157.185) xenial; urgency=medium
 
   * linux: 4.4.0-157.185 -proposed tracker (LP: #1837476)
-- 
2.7.4

