From 4ad0d922a3f4004e9055aa9bfe32a658568c1871 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Mon, 20 Apr 2020 04:20:00 -0500
Subject: [PATCH 2/6] Changelog

---
 debian.master/changelog | 48 ++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 48 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index 3768bc1..57b379e 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,51 @@
+linux (4.4.0-177202004200420.0+mediatree+hauppauge) xenial; urgency=low
+
+  * Ubuntu-master kernel git clone
+    - 4.4.0-177.207 rev 1d6d038afa709652a3537ce886d1929d86f22a89
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2019-10-11-12-13.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 07.si2157.analog.tuning
+    --- 0001-si2157-Enable-tuner-status-flags.patch
+    --- 0002-si2157-Check-error-status-bit-on-cmd-execute.patch
+    --- 0003-si2157-Better-check-for-running-tuner-in-init.patch
+    --- 0004-si2157-Add-analog-tuning-related-functions.patch
+    --- 0005-si2157-Briefly-wait-for-tuning-operation-to-complete.patch
+    --- 0006-cx23885-Add-analog-tuner-support-to-Hauppauge-QuadHD.patch
+    --- 0007-cx23885-Add-analog-frontend-to-1265_K4.patch
+    --- 0008-cx231xx-Add-i2c-device-analog-tuner-support.patch
+    --- 0009-cx23885-Add-i2c-device-analog-tuner-support.patch
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0004-si2168-different-default-that-windows-driver.patch
+    --- 0005-Ignore-first-cold-command-error-for-si2141-in-_init.patch
+    - 20.missing.stat.bits
+    --- 0001-si2157-add-on-demand-rf-strength-func.patch
+    --- 0002-lgdt3306a-Add-CNR-v5-stat.patch
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+    - 80.montage.3103b
+    --- 0001-m88ds3103-Add-Montage-3103b-support.patch
+    --- 0002-em28xx-Add-Hauppauge-461e-revision-2.patch
+    --- 0003-dw2102-Log-probed-demodulator-type-and-address.patch
+    --- 0004-dw2102-probe-for-demodulator-i2c-address.patch
+    --- 0005-dw2102-check-initialization-replies.patch
+    --- 0006-dw2102-warn-on-failed-i2c-read-requests.patch
+    - 90.usb.quadhd.xml692
+    --- 0001-mxl69x-MaxLinear-692-ATSC-demod-tuner-driver.patch
+    --- 0002-em28xx-core-Fix-TS2-active-led.patch
+    --- 0003-em28xx-Add-support-for-Hauppauge-USB-QuadHD.patch
+
+ -- Fake User <hidden@email.co>  Mon, 20 Apr 2020 04:20:00 -0500
+
 linux (4.4.0-177.207) xenial; urgency=medium
 
   * xenial/linux: 4.4.0-177.207 -proposed tracker (LP: #1867243)
-- 
2.7.4

