From 3ba7be30bfe89a309a17b4ee9da17b42ccb68a1e Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Tue, 6 Apr 2021 16:20:00 -0500
Subject: [PATCH 2/6] Changelog

---
 debian.master/changelog | 39 +++++++++++++++++++++++++++++++++++++++
 1 file changed, 39 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index b980a6c..8edf18e 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,42 @@
+linux (4.4.0-206202104061620.0+mediatree+hauppauge) xenial; urgency=low
+
+  * Ubuntu-master kernel git clone
+    - 4.4.0-206.238 rev c4c858a539eaefb9ff7a2433118653ad1cc78946
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2020-10-10-10-10.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0004-si2168-different-default-that-windows-driver.patch
+    --- 0005-dw2102-check-initialization-replies.patch
+    --- 0006-dw2102-warn-on-failed-i2c-read-requests.patch
+    --- 0007-si2168-Include-b-4.0-11-when-checking-firmware-versi.patch
+    --- 0008-Fix-saa7164-REV2-firmware.patch
+    --- 0009-em28xx-Add-pid-for-bulk-revision-of-Hauppauge-461e.patch
+    --- 0010-em28xx-Add-pid-for-bulk-revision-of-Hauppauge-461eV2.patch
+    --- 0099-media-rc-ensure-that-uevent-can-be-read-directly-after-rc-device-register.patch
+    - 11.cx23885.patches
+    --- 0001-cx2885-fix-analog-video-routings.patch
+    --- 0002-cx23885-module-option-to-disable-analog-video.patch
+    --- 0003-cx23885-add-more-ryzen-pci-id-to-dma-override-list.patch
+    - 20.missing.stat.bits
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+    - 90.usb.quadhd.mxl692
+    --- 0001-mxl692-MaxLinear-692-ATSC-demod-tuner-driver.patch
+    --- 0002-em28xx-core-Fix-TS2-active-led.patch
+    --- 0003-em28xx-Add-support-for-Hauppauge-USB-QuadHD.patch
+
+ -- Fake User <hidden@email.co>  Tue, 06 Apr 2021 16:20:00 -0500
+
 linux (4.4.0-206.238) xenial; urgency=medium
 
   * xenial/linux: 4.4.0-206.238 -proposed tracker (LP: #1919242)
-- 
2.7.4

