From 7c5147723c0485a0d61188acf1482eecb8804729 Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Fri, 4 Sep 2020 10:10:10 -0500
Subject: [PATCH 2/6] Changelog

---
 debian.master/changelog | 30 ++++++++++++++++++++++++++++++
 1 file changed, 30 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index 222e2ff..91b7200 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,33 @@
+linux (4.4.0-189202009041010.0+mediatree+hauppauge) xenial; urgency=low
+
+  * Ubuntu-master kernel git clone
+    - 4.4.0-189.219 rev def589a8bcb18f63b7c2d73726e83a704e55c768
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2020-07-07-07-07.patch
+    - 0002-Changelog.patch
+    - 0003-Apply-build-fixes-to-media-tree.patch
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+    - 0005-Add-new-media-tree-kernel-config.patch
+    - 0006-Packaging-updates.patch
+
+  * Additional patches
+    - 10.random.patches
+    --- 0001-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    --- 0002-cx23885-Set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    --- 0003-si2168-extend-i2c-timeout-insert-cmd-status-poll-del.patch
+    --- 0004-si2168-different-default-that-windows-driver.patch
+    --- 0005-dw2102-check-initialization-replies.patch
+    --- 0006-dw2102-warn-on-failed-i2c-read-requests.patch
+    - 20.missing.stat.bits
+    --- 0003-si2168-Add-some-various-v3-stat-ioctls.patch
+    - 90.usb.quadhd.xml692
+    --- 0001-mxl69x-MaxLinear-692-ATSC-demod-tuner-driver.patch
+    --- 0002-em28xx-core-Fix-TS2-active-led.patch
+    --- 0003-em28xx-Add-support-for-Hauppauge-USB-QuadHD.patch
+
+ -- Fake User <hidden@email.co>  Fri, 04 Sep 2020 10:10:10 -0500
+
 linux (4.4.0-189.219) xenial; urgency=medium
 
   * xenial/linux: 4.4.0-189.219 -proposed tracker (LP: #1891057)
-- 
2.7.4

