From ae0189f9d66afbda047bc2db912c4aab7cbaa1ef Mon Sep 17 00:00:00 2001
From: Brad Love <hidden@email.co>
Date: Tue, 20 Jun 2017 06:20:11 -0500
Subject: [PATCH 6/6] Changelog

---
 debian.master/changelog | 27 +++++++++++++++++++++++++++
 1 file changed, 27 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index bf5a70a..4ded1f8 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,30 @@
+linux (4.4.0-81201706200620.0+mediatree+hauppauge) xenial; urgency=low
+
+  * Ubuntu kernel git clone
+    - 4.4.0-81.104 rev 78ab29f4aa24e326204fd6200666d7772100b08c
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2017-06-20-04-20.patch
+    - 0002-Apply-build-fixes-to-media-tree.patch
+    - 0003-Add-new-media-tree-kernel-config.patch
+
+  * Ubuntu mainline build fixes
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+
+  * Packaging patches
+    - 0005-Packaging-updates.patch
+    - 0006-Changelog.patch
+
+  * Additional patches
+    - 0001-lgdt3306a-v4.6-i2c-mux-backport.patch
+    - 0002-em28xx-Hauppauge-DualHD-DVB-ATSC-second-tuner-suppor.patch
+    - 0003-em28xx-bulk-transfer-implementation-fix.patch
+    - 0004-em28xx-usb-packet-size-tweaks.patch
+    - 0005-lgdt3306a-add-QAM-AUTO-support.patch
+    - 0006-em28xx-Add-Hauppauge-SoloHD-DualHD-bulk-models-suppo.patch
+
+ -- Fake User <hidden@email.co>  Tue, 20 Jun 2017 06:20:10 -0500
+
 linux (4.4.0-81.104) xenial; urgency=low
 
   * CVE-2017-1000364
-- 
2.7.4

