From b85166d41e548019b7707df4907b316ee5632b99 Mon Sep 17 00:00:00 2001
From: root <hidden@email.co>
Date: Mon, 11 Dec 2017 19:47:56 +0000
Subject: [PATCH 6/6] Changelog

---
 debian.master/changelog | 45 +++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 45 insertions(+)

diff --git a/debian.master/changelog b/debian.master/changelog
index 47fc1fe..0273b95 100644
--- a/debian.master/changelog
+++ b/debian.master/changelog
@@ -1,3 +1,48 @@
+linux (4.4.0-103201712111947.0+mediatree+hauppauge) xenial; urgency=low
+
+  * Ubuntu kernel git clone
+    - 4.4.0-103.126 rev 2c5158e82d497c5eb90d6e2b8aaf07d36cb175f6
+
+  * LinuxTV.org media tree slipstream + build fixes
+    - 0001-Linuxtv.org-media-tree-sync-2017-10-05-20-10.patch
+    - 0002-Apply-build-fixes-to-media-tree.patch
+    - 0003-Add-new-media-tree-kernel-config.patch
+
+  * Ubuntu mainline build fixes
+    - 0004-Mainline-Ubuntu-build-fixes.patch
+
+  * Packaging patches
+    - 0005-Packaging-updates.patch
+    - 0006-Changelog.patch
+
+  * Additional patches
+    - 0001-lgdt3306a-v4.6-i2c-mux-backport.patch
+    - 0002-em28xx-Hauppauge-DualHD-DVB-ATSC-second-tuner-suppor.patch
+    - 0003-em28xx-bulk-transfer-implementation-fix.patch
+    - 0004-em28xx-usb-packet-size-tweaks.patch
+    - 0005-lgdt3306a-add-QAM-AUTO-support.patch
+    - 0006-em28xx-Add-Hauppauge-SoloHD-DualHD-bulk-models-suppo.patch
+    - 0007-cx23885-Enable-new-Hauppauge-PCIe-ImpactVCB-variant.patch
+    - 0008-cx23885-set-Hauppauge-HVR1255-tuner-type-to-TDA18271.patch
+    - 0009-si2157-check-error-status-bit-on-cmd-execute.patch
+    - 0010-si2157-briefly-wait-for-tuning-operation-to-complete.patch
+    - 0011-Si2157-better-check-for-running-tuner-in-init-Reset.patch
+    - 0012-si2157-add-analog-tuner-support-and-dvb-frontend.patch
+    - 0013-cx23xxx-Ensure-i2c-platform-data-object-lifetime.patch
+    - 0014-cx23885-add-support-for-Hauppauge-HVR1265-K4.patch
+    - 0015-cx25840-remove-offset-from-chip-specific-registers-A.patch
+    - 0016-cx28335-Add-support-for-Hauppauge-Starburst-2-PCIe.patch
+    - 0017-cx23885-Add-support-for-new-Hauppauge-QuadHD-885.patch
+    - 0018-cx23885-Add-analog-tuner-support-to-Hauppauge-QuadHD.patch
+    - 0019-cx23885-fix-gpio-on-Hauppauge-QuadHD-888-PCIe-cards.patch
+    - 0020-lgdt3306a-QAM-streaming-improvement-Required-for.patch
+    - 0021-si2168-spectral-inversion-property-Hauppauge.patch
+    - 0022-si2168-Change-from-DVB-T-to-DVB-T-T2-autodetect.patch
+    - 0023-em28xx-Enable-Hauppauge-SoloHD-rebranded-292e-SE.patch
+    - 0024-em28xx-increase-max-em28xx-boards-to-max-dvb-adapter.patch
+
+ -- Fake User <hidden@email.co>  Mon, 11 Dec 2017 19:47:56 +0000
+
 linux (4.4.0-103.126) xenial; urgency=low
 
   * linux: 4.4.0-103.126 -proposed tracker (LP: #1736181)
-- 
2.7.4

